modules/mixcr/1.0/config/default.yaml

```diff
@@ -16,12 +17,17 @@ lcr-modules:
                     --starting-material rna
                 genome:
                     --starting-material dna
+
+        conda_envs: 
+            java: "{MODSDIR}/envs/java-jdk-8.0.yaml"
             
         threads:
             mixcr_run: 4
 
-        mem_mb:
-            mixcr_run: 6000
+        resources:
+            mixcr_run: 
+                mem_mb: 6000
+                fastq: 1
```

Comment:
I wonder if this should be the default of 2 for fastq1 and fastq2?

Reply:
I think of it as the equivalent of the single bam file, e.g. how many samples I want to run at one time.

Reply:
I see, so when you run you specify fastq as n of samples * 2?

Reply:
Well no, because this value really specifies how many instances of this rule will be run simultaneously, and since each run utilizes 2 fastqs, if you specify `--resources fastq=10` on the command line, it will use 20 fastqs at a time.